lib/hope-tab.js

```diff
@@ -123,9 +123,9 @@ window.HopeTab = fxosComponent.register('hope-tab', {
         </div>
         <div class="url">
           <div class="field">
-            <button class="reader"></button>
+            <button class="reader" data-icon="more" aria-label="more" ></button>
             <div class="url-text"></div>
-            <button class="refresh"></button>
+            <button class="refresh" data-icon="reload" aria-label="reload" ></button>
```

Comment:
Already was being loaded:
https://github.com/etiennesegonzac/hope/blob/995f01832fd85562dbbb14c3d0fecb07108c39d0/index.html#L7

I have just added it for all other icons etc.